File: app/src/main/java/com/scatl/uestcbbs/module/darkroom/view/DarkRoomView.java
package com.scatl.uestcbbs.module.darkroom.view;

import com.scatl.uestcbbs.entity.DarkRoomBean;

import java.util.List;

/**
 * author: sca_tl
 * date: 2021/4/10 12:07
 * description:
 */
public interface DarkRoomView {
    void onGetDarkRoomDataSuccess(List<DarkRoomBean> darkRoomBeanList);
    void onGetDarkRoomDataError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/darkroom/view/DarkRoomView.kt
package com.scatl.uestcbbs.module.darkroom.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.DarkRoomBean

/**
 * Created by sca_tl at 2023/6/6 15:32
 */
interface DarkRoomView: BaseView {
    fun onGetDarkRoomDataSuccess(darkRoomBeanList: List<DarkRoomBean>)
    fun onGetDarkRoomDataError(msg: String?)
}